requirements.sh

```diff
@@ -25,18 +25,9 @@ set -e
 # Apt package
 sudo apt-get install libusb-1.0-0-dev
 
-mkdir /tmp/wind-turbine-install
-cd /tmp/wind-turbine-install
-
-# Linux specific prerequisites - https://www.phidgets.com/docs/OS_-_Linux
-wget https://www.phidgets.com/downloads/phidget22/libraries/linux/libphidget22.tar.gz
-tar -xzvf /tmp/wind-turbine-install/libphidget22.tar.gz
-cd /tmp/wind-turbine-install/libphidget22-*
-./configure --prefix=/ && make && sudo make install
-fn=`find -name *libphidget22.rule*`
-sudo mv ${fn} /etc/udev/rules.d.
+## extras install
+## ./extras_install
```

Comment:
Uncomment the code line; as we need this when non-package installation is required.